<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startServerButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="connectionTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Client ID</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Count of connections</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="conectionsCountLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="reserveServerlineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reserve Server</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LogWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>311</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1:1221</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startServerButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Server</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
